mux/bindings/generate.sh

```
@@ -0,0 +1,99 @@
+  local tmp_rel
+  local prompt
+
+  tmp="$(mktemp -d "$SCRIPT_DIR/.tmp-$lang.XXXXXX")"
+  tmp_rel="${tmp#$REPO_ROOT/}"
+  prompt="$(assemble_prompt "$lang" "$tmp_rel")"
+
+  echo "Generating $lang binding into temporary directory $tmp_rel"
+  if codex exec --cd "$REPO_ROOT" "$(cat "$prompt")"; then
+    rm -rf "$out"
+    mv "$tmp" "$out"
+    echo "Installed $lang binding into ${out#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)